example/mediaelement-webaudio/app.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+// Create an instance
+var wavesurfer = {};
+
+// Init & load audio file
+document.addEventListener('DOMContentLoaded', function() {
+    // Init
+    wavesurfer = WaveSurfer.create({
+        container: document.querySelector('#waveform'),
+        backend: 'MediaElementWebAudio',
+        minPxPerSec: 30,
+        scrollParent: true,
+        waveColor: '#A8DBA8',
+        progressColor: '#3B8686',
+        plugins: [
+            WaveSurfer.timeline.create({
+                container: '#timeline'
+            })
+        ]
+    });
+
+    // get audio peaks
+    fetch('peaks.json')
+        .then(response => {
+            return response.json();
+        })
+        .then(peaks => {
+            let max = peaks.data.reduce((max, el) => (el > max ? el : max));
+
+            return peaks.data.map(el => {
+                return el / max;
+            });
+        })
+        .then(normalizedPeaks => {
+            // Load audio from URL
+            wavesurfer.load('stereo.mp3', normalizedPeaks, 51);
```

Comment:
you should load the file from `../media/`. If it's a copy, adjust both examples to use the same file.